tests/quality_pragmaitc/test_metric.py

```diff
@@ -1,20 +1,6 @@
-from goald.quality.pragmatic.model.metric import MetricsList
-from goald.quality.pragmatic.model.metric import Metric
-from goald.quality.pragmatic.model.metric import CommonMetrics
+from goald.quality.pragmatic.model.common_metrics import CommonMetrics
 
-def test_metric_list():
-    metricsList = MetricsList()
-
-    METERS = Metric('METERS', False)
-    SECONDS = Metric('SECONDS', True)
-
-    metricsList.setMetricsList(METERS, SECONDS)
-
-    assert metricsList.getLessIsBetter(METERS) == False
-    assert metricsList.getLessIsBetter(SECONDS) == True
 
 def test_common_metrics():
-    commonMetrics = CommonMetrics()
-
-    assert commonMetrics.METERS.getLessIsBetter() == False
-    assert commonMetrics.SECONDS.getLessIsBetter() == True
+    assert False is CommonMetrics.METERS.getLessIsBetter() 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)